outputs/prometheus_output/prometheus_output.go

```diff
@@ -121,7 +121,6 @@ func (p *PrometheusOutput) SetEventProcessors(ps map[string]map[string]interface
 func (p *PrometheusOutput) Init(ctx context.Context, cfg map[string]interface{}, opts ...outputs.Option) error {
 	err := outputs.DecodeConfig(cfg, p.Cfg)
 	if err != nil {
-		p.logger.Printf("prometheus output config decode failed: %v", err)
```

Comment:
there will be no message in the log that output failed now?
maybe we should process log option first, which will init the logger, and then we will be able to handle logging of the error message?

Reply:
there will be a log message on the caller of the Init function

Reply:
will fix, option dont use the config, so should be fine